### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by Galithil**
Better to merge Mario's PR first.

Changes number of cores to 8 by default.

### CODE REVIEW

File: ngi_pipeline/engines/piper_ngi/launchers.py [hunk 1]
```diff
@@ -216,7 +216,7 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
         raise RuntimeError('No SLURM project id specified in configuration file '
                            'for job "{}"'.format(job_identifier))
     slurm_queue = config.get("slurm", {}).get("queue") or "node"
-    num_cores = config.get("slurm", {}).get("cores") or 16
+    num_cores = config.get("slurm", {}).get("cores") or 8
```
Comment (guillermo-carrasco) [inline ngi_pipeline/engines/piper_ngi/launchers.py]: I didn't know this syntax, cool!

Comment (pekrau) [inline ngi_pipeline/engines/piper_ngi/launchers.py]: Being picky: Shouldn't that "8" be in a config file somewhere? Or at least as a symbol, defined as DEFAULT_NUMBER_CORES=8 somewhere.

Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/launchers.py]: Yeah it should be but if the user doesn't specify in the config file we default to 8

Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/launchers.py]: So it would be in the config file like
```yaml
slurm:
    cores: 8
```

**Pull request merged**